downloads/id/github-git-advanced-cheat-sheet.md

```diff
@@ -0,0 +1,67 @@
+# Buku Saku Tingkat Lanjut
+
+## Perintah penggabungan
+
+---
+
+Batalkan penggabungan
+
+`$ git merge --abort`
+
+## Perintah penyimpanan
+
+---
+
+Simpan perubahan saat ini dalam penyimpanan dengan nama tertentu
+
+`$ git stash save <stash name>`
+
+Simpan perubahan saat ini dalam sebuah penyimpanan (dengan nama stash@{0})
+
+`$ git stash`
+
+Hilangkan simpanan di tumpukan teratas
+
+`$ git stash drop`
+
+Hilangkan simpanan di indeks tumpukan ke-n
+
+`$ git stash drop stash@{n}`
+
+Terapkan penyimpanan di indeks tumpukan ke-n dan hapus dari daftar
+
+`$ git stash pop stash@{n}`
+
+Terapkan penyimpanan di indeks tumpukan ke-n
+
+`$ git stash apply stash@{n}`
+
+## Perintah lapor keluar
+
+---
+
+Singkirkan semua perubahan
+
+`$ git checkout .`
+
+Membuat cabang baru dan pindah ke cabang tersebut
+
+`$ git checkout -b <branch name>`
+
+Membawa satu berkas ke lokasi saat ini dari simpanan
+
+`$ git checkout <stash@{0}> — <filename>`
```

Comment:
"filename" is bahasa Indonesian?  "nama berkas", but I believe that simply "berkas" is also acceptable in this case.  I am not sure which is preferable.